project/Dependencies.scala

```diff
@@ -19,7 +19,7 @@ object Dependencies {
     val socrataCuratorUtils = "1.2.0"
     val socrataThirdPartyUtils = "5.0.0"
     val socrataHttpCuratorBroker = "3.13.4"
-    val soqlStdlib = "4.0.1"
+    val soqlStdlib = "4.0.3"
```

Comment:
have you verified with author of 4.0.2 that it's ok to bump this? (i see no reason not, but just to be safe!)

Reply:
Yup! Chatted with Chi and he said it is good to go